backend/qemu.pm

```diff
@@ -186,14 +186,15 @@ sub do_extract_assets {
     my $hdd_num = $args->{hdd_num};
     my $name    = $args->{name};
     my $img_dir = $args->{dir};
-    my $format  = $args->{format};
+    $name =~ /\.([[:alnum:]]+)$/;
+    my $format = $1;
```

Comment:
Hi @coolo, this binding operation is to let $1 get format in $name, and $name still have format. The code just look like what I modified before PR 415. So I got the $format in that way is because I'm not sure if getting format from $args have problem or not (since it didn't use this way before..)